test/definitions/features/installer_test.rb

```diff
@@ -44,7 +45,7 @@
       it '#upgrade runs the installer with correct params' do
         assume_feature_absent(:satellite)
         installer_inst.expects(:'execute!').
-          with('LANG=en_US.utf-8 foreman-installer --disable-system-checks --upgrade',
+          with("LANG=en_US.utf-8 foreman-installer #{subject.installer_arguments}",
```

Comment:
Do you really need to call `installer_arguments` method here? 
In before block you are already stubbing this method to return `--disable-system-checks` & `--upgrade` flag.